coordinator/src/main/scala/filodb/coordinator/ProtoConverters.scala

```diff
@@ -1311,6 +1313,26 @@ object ProtoConverters {
     }
   }
 
+  implicit class RemoteActorPlanDispatcherToProtoConverter(apd: filodb.coordinator.RemoteActorPlanDispatcher) {
+    def toProto(): GrpcMultiPartitionQueryService.RemoteActorPlanDispatcher = {
+      val builder = GrpcMultiPartitionQueryService.RemoteActorPlanDispatcher.newBuilder()
+      builder.setPlanDispatcher(apd.asInstanceOf[filodb.query.exec.PlanDispatcher].toProto)
```

Comment:
I'm ultra late to ask this, so it should be deferred to later work if needed. But could we use plain all-argument constructor calls here to give compile-time errors when a `to/fromProto` needs to be updated?

Reply:
yep, let's do it for all classes in one swing later, pretty much all the converter methods now are like the above